packages/@aws-cdk/custom-resources/README.md

```diff
@@ -101,3 +101,6 @@ const awsCustom2 = new AwsCustomResource(this, 'API2', {
   }
 })
 ```
+
+##### Booleans conversion
+Since custom resource properties are always stringified by CloudFormation, `true`/`false` values will automatically be converted back to booleans. If the strings `'true'`/`'false'` are required in the SDK call, use `'true!'`/`'false!'`.
```

Comment:
Wait, can’t we convert Boolean true/false to their encoded values in the construct?

Reply:
Passing either `true`/`false` or `'true'`/`'false'` in the custom resource properties will end up as a string in the event received by the Lambda function. This conversions needs to be handled in the Lambda.

Reply:
OK, you mean the other way around? Let the user specify "normal" values and encode them in the construct as special strings to be able to decode in the Lambda?